themes/question/bootstrap_dropdown/survey/questions/answer/list_dropdown/answer.twig

```diff
@@ -51,15 +51,19 @@
         $(function () {
             var selectPicker = $('#answer{{ name }}').selectpicker();
 
-            // Custom js to make width setting work with this beta version:
+            {#
+                Custom js to make width setting work with this beta version.
+                The bootstrap-select's auto-width feature relies on a handler set for 'loaded.bs.select' by setWidth(),
+                which is called after the event is triggered, so the auto-width's handler is never executed.
+                This is a workaround to make the auto-width work.
+            #}
             {% if width_entry != '' %}
-            selectPicker.on('loaded.bs.select', function () {
-                selectPicker.selectpicker('open');
-                var dropdown = selectPicker.parent();
-                var width = dropdown.find('.dropdown-menu.show').width();
-                dropdown.css('width', width);
-                selectPicker.selectpicker('toggle');
-                $('.bootstrap-select .btn.dropdown-toggle').blur();
+            selectPicker.on('loaded.bs.select.firstload', function () {
```

Comment:
We don't have issue if dropdown start hidden by condition and shown after ?

Reply:
Well, not sure... We are using the same event as before (just a custom one), so don't expect issues.
Could be a good testing scenario @tiborpacalat